src/PhpParser/Node/Manipulator/ClassManipulator.php

```diff
@@ -141,6 +143,15 @@ public function addConstructorDependencyWithCustomAssign(
 
         $this->addParameterAndAssignToMethod($constructorMethod, $variableInfo, $assign);
 
+        /** @var string|null $parentClassName */
+        $parentClassName = $classNode->getAttribute(AttributeKey::PARENT_CLASS_NAME);
+
+        // Symfony Command, special case to add "parent::__construct()"
+        if ($parentClassName && Strings::endsWith($parentClassName, 'Command')) {
```

Comment:
This is hardly bound to Symfony Console call, so I'm honestly not sure this should be placed here.

Can't we check on `ContainerGetToConstructorInjectionRector` that parent class (or one of parents) is `Symfony\Component\Console\Command\Command`, and if so, force injection of `parent::__construct()` call?

Also there is another bug I guess when you have class extending parent class with `__construct()` call, now it will not force adding it, no? (at least it looks like that ;) )

Reply:
> Can't we check on ContainerGetToConstructorInjectionRector (similarly to job done by MakeCommandLazyRector) that parent class (or one of parents) is Symfony\Component\Console\Command\Command, and if so, force injection of parent::__construct() call?

:+1: 

> Also there is another bug I guess when you have class extending parent class with __construct() call, now it will not force adding it, no? (at least it looks like that ;) )

What do you mean? Could you share code snippet?

Reply:
```php
<?php

use Rector\Symfony\Tests\FrameworkBundle\ContainerGetToConstructorInjectionRector\Source\ContainerAwareParentWithConstructClass;

final class SomeClass extends \Rector\Symfony\Tests\FrameworkBundle\ContainerGetToConstructorInjectionRector\Source\ContainerAwareParentWithConstructClass
{
    protected function doSomethingFancy()
    {
        $someService = $this->getContainer()->get('translator');

        $someService = $this->getContainer()->get('translator')->translateSomething();
    }
}

?>
-----
<?php

use Rector\Symfony\Tests\FrameworkBundle\ContainerGetToConstructorInjectionRector\Source\ContainerAwareParentWithConstructClass;

final class SomeClass extends  \Rector\Symfony\Tests\FrameworkBundle\ContainerGetToConstructorInjectionRector\Source\ContainerAwareParentWithConstructClass
{
    /**
     * @var \Rector\Symfony\Tests\Rector\FrameworkBundle\AbstractToConstructorInjectionRectorSource\SomeTranslatorInterface
     */
    private $someTranslator;
    public function __construct(\Rector\Symfony\Tests\Rector\FrameworkBundle\AbstractToConstructorInjectionRectorSource\SomeTranslatorInterface $someTranslator)
    {
        parent::__construct();
        $this->someTranslator = $someTranslator;
    }
    protected function doSomethingFancy()
    {
        $someService = $this->someTranslator;

        $someService = $this->someTranslator->translateSomething();
    }
}

?>
```
And extended class:
```php
<?php declare(strict_types=1);

namespace Rector\Symfony\Tests\FrameworkBundle\ContainerGetToConstructorInjectionRector\Source;
;
use Symfony\Component\DependencyInjection\ContainerInterface;

class ContainerAwareParentWithConstructClass
{
    public function __construct()
    {
        $this->fakeCall();
    }

    public function getContainer(): ContainerInterface
    {
    }
    
    private function fakeCall()
    {
        
    }
}
```
I must admit that I didn't validate this locally ;)

Reply:
I see :)

Reply:
See last commit